pkg/cmd/cli/secrets/dockercfg.go

```diff
@@ -0,0 +1,197 @@
+package secrets
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/credentialprovider"
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	CreateDockerConfigSecretRecommendedName = "new-dockercfg"
+
+	createDockercfgLong = `Create a new dockercfg secret
+
+Dockercfg secrets are used to authenticate against docker registries.
+
+When using plain docker, you can authenticate to a given registry by running 
```

Comment:
It should be capitalized when we refer to Docker the command line client.  Also, "plain docker" is awkward.  "When using the Docker command line to push images, you can ..."

----- Original Message -----
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
> > +	"github.com/GoogleCloudPlatform/kubernetes/pkg/credentialprovider"
> > +	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
> > +
> > +	"github.com/spf13/cobra"
> > +)
> > +
> > +const (
> > +	CreateDockerConfigSecretRecommendedName = "new-dockercfg"
> > +
> > +	createDockercfgLong = `Create a new dockercfg secret
> > +
> > +Dockercfg secrets are used to authenticate against docker registries.
> > +
> > +When using plain docker, you can authenticate to a given registry by
> > running
> 
> @smarterclayton - do we need to capitalize Docker versus docker?  Is it only
> Docker when referring to the company and not the tool?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2866/files#r32044994